Microsoft/WebTemplateStudio
# Azure names - trim generated name that are too long

This is required for always generating names (fields not exposed to the users) that are within the maximum length for each services.

**To test:**
1. Launch extension in preview mode
2. Enter a really long project name
3. Go to the Azure page and add any services
4. When the form opens up, error should show up and you'll have to shorten the name.
5. Add App Service to make sure that the generated name for App Service Plan (not exposed to user) is valid
